permissions/plugin-qaconsole-test-automation-reporter.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "qaconsole-test-automation-reporter"
+github: "jenkinsci/qaconsole-test-automation-reporter-plugin"
+paths:
+  - "org/jenkins-ci/plugins/qaconsole-test-automation-reporter"
```

Comment:
This actually needs to be "com/qaconsole/qaconsole-test-automation-reporter-plugin" since that is the groupId in your pom.xml: https://github.com/jenkinsci/qaconsole-test-automation-reporter-plugin/blob/master/pom.xml#L10.